File: grad_project-main/gradu/app/Models/Category.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Category extends Model
{
    use HasFactory;
    protected $table = 'menu_categories';
    protected $fillable = [
        'category_title',
        'category_icon'
    ];
}

File: kitchen/lib/src/Models/Category.dart
class Category{
  int? id;
  String? category_title;
  String? category_icon;

  Category({this.id, this.category_title, this.category_icon});

  factory Category.fromJson(Map<String, dynamic> json){
    return Category(
      id: json['id'],
      category_title: json['category_title'],
      category_icon: json['category_icon']
    );
  }
}